src/nomad_analysis/general/schema.py

```diff
@@ -16,11 +16,36 @@
 # limitations under the License.
 #
 
+from nomad.datamodel.metainfo.basesections import (
+    ActivityStep,
+    Entity,
+)
 from nomad.metainfo import (
     SchemaPackage,
+    SubSection,
 )
 
 m_package = SchemaPackage()
 
 
+class AnalysisResult(Entity):
```

Comment:
I guess we will rework this later to the Correlated Properties but this is fine for now.

Reply:
Sounds good!